conf/application.conf

```diff
@@ -6,6 +6,6 @@ auth.domain="auth.tdr-integration.nationalarchives.gov.uk"
 
 upload.url="https://upload.tdr-integration.nationalarchives.gov.uk"
 
-consignmentapi.domain="https://api.tdr-integration.nationalarchives.gov.uk"
+consignmentapi.domain="http://localhost:8080"
```

Comment:
Should this be change to the `localhost`. This configuration is used for local development, and I think generally devs will want to point to intg api.